webviews/src/kanban/shared/styles.css

```
@@ -0,0 +1,254 @@
+  padding: 10px;
+  border-radius: 9px;
+  border: 1px solid var(--agent-border, rgba(128, 128, 128, 0.22));
+  background: var(--agent-surface, rgba(128, 128, 128, 0.06));
+  box-shadow: 0 1px 2px var(--agent-shadow, rgba(0, 0, 0, 0.08));
+}
+
+.kanban-card__title {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 1.3;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)